src/script-child-process.ts

```diff
@@ -119,10 +188,74 @@ export class ScriptChildProcess {
         } else {
           resolve({ok: true, value: undefined});
         }
+        this.#state = 'stopped';
       });
     });
   }
 
+  /**
+   * Mark this child process for termination. On Linux/macOS, sends a `SIGINT`
+   * signal. On Windows, invokes `taskkill /pid PID /t /f`.
```

Comment:
Perhaps the comment after the first sentence belongs on the `#actuallyTerminate` method?

Reply:
I was thinking that since this is the public one, it's more useful to put the details there, so that you can see it when you hover-over from another module.

Reply:
Ooh, okay that makes sense!